app/src/androidTest/java/com/xa/xpensauditor/AccountSettingsTest.java

```diff
@@ -124,13 +126,13 @@ public void accountSettingsTest() {
                         withParent(withParent(IsInstanceOf.<View>instanceOf(android.view.ViewGroup.class))),
                         isDisplayed()));
         button.check(matches(isDisplayed()));
-
+//Changing password
         ViewInteraction button2 = onView(
                 allOf(withId(R.id.change_password_button), withText("CHANGE PASSWORD"),
                         withParent(withParent(IsInstanceOf.<View>instanceOf(android.view.ViewGroup.class))),
                         isDisplayed()));
         button2.check(matches(isDisplayed()));
-
+//sending the password reset mail
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Comment has incorrect indentation level 0, expected is 8, indentation should be the same level as line 136.